modules/search/class.jetpack-search.php

```diff
@@ -251,6 +251,7 @@ public function load_assets() {
 					'postTypeFilters'    => $widget_options['post_types'],
 					'postTypes'          => $post_type_labels,
 					'siteId'             => Jetpack::get_option( 'id' ),
+					'siteUrl'            => site_url(),
```

Comment:
Should it be `home_url()` here instead, to avoid breaking sites where WordPress is in a subdirectory?
https://wordpress.org/support/article/giving-wordpress-its-own-directory/

Here is an example:
https://yellowed-canid.jurassic.ninja/

Reply:
✅ Updated to use `home_url()` instead of `site_url()`